news-sentiment/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Analyze the Sentiment of News Headlines
+description: A Google Sheet with Apps Script that Analyzes the sentiment of news headlines retrieved for a user specified topic from a free News API.
+labels: Apps Script, Sheets, Cloud Natural Language
+material_icon: Feed
+create_time: 2021-03-22
+update_time: 2021-03-22
+---
+
+This solution consists of a Google Sheet that uses Apps Script to fetch the current news headlines from the free news API `newsapi.org` using a user provded `topic`.
+
+Once the news headlines are fetched and loaded onto the sheet, it runs a Natural Language sentiment detection for each headline. 
+As the sentiments are returned from the Google Cloud Language API, the sheet is updated with both a numeric score as well as a `happy`, `meh` or `sad` icon depending on the sentiment.
+ 
+## Technology highlights
+
+- The sheet provides an extra menu option: **News Headlines Sentiments** and when clicked renders the option: **Analyze News headlines...**.  When selected, it calls the `showNewsPrompt()` function and triggers a popup dialog to allow the user to enter their own news topic of choice. 
```

Comment:
Not sure the menu is highlight worthy, and in general don't need to be this specific here -- folks can look at the code to see how it works. Just hit the highlights, something like:

* Shows how to add custom menus to the sheets UI
* Uses UrlFetch to load news headlines from an external API
* Demonstrates the use of the Cloud Natural Language API to analyze headlines.

Reply:
Done. I simplified this section down to what you recommended.. Much simpler..